src/python/pants/option/options.py

```diff
@@ -317,14 +317,25 @@ def for_scope(self, scope, inherit_from_enclosing_scope=True):
 
     return values
 
-  def get_fingerprintable_for_scope(self, scope):
+  def get_fingerprintable_for_scope(self, scope, include_passthru=False):
```

Comment:
Do we need the new arg? Can we just always fprint the result of `self.passthru_args_for_scope()` which will be an empy list if the task doesn't support them?